source/faq.rst

```diff
@@ -0,0 +1,98 @@
+.. _faq:
+
+================================================================================
+Frequently Asked Questions
+================================================================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+Why do we need to follow "the Kubernetes way" when Kubernetes was never intended to run databases?
+=====================================================================================================
+
+As it is well known, the Kubernetes approach is targeted at stateless
+applications but provides ways to store state (in Persistent Volumes, etc.) if
+the application needs it. Generally, a stateless mode of operation is supposed
+to provide better safety, sustainability, and scalability, it makes the
+already-deployed components interchangeable. You can find more about substantial
+benefits brought by Kubernetes to databases in `this blog post <https://www.percona.com/blog/2020/10/08/the-criticality-of-a-kubernetes-operator-for-databases/>`_.
+
+The architecture of state-centric applications (like databases) should be
+composed in a right way to allow avoiding crashes, data loss, or data
+inconsistencies during hardware fail. It is the main goal of the database
+Kubernetes Operator to solve or simplify these aspects.
```

Comment:
The architecture of state-centric applications (like databases) should be
composed in a right way to avoid crashes, data loss, or data
inconsistencies during hardware failure. Percona Kubernetes Operator for Percona XtraDB Cluster provides out-of-the-box functionality to automate provisioning and management of highly available MySQL database clusters on Kubernetes.